src/deploy-events.js

```diff
@@ -1,15 +1,19 @@
-const fs = require('node:fs');
-const path = require('node:path');
+const guildScheduledEventCreate = require('./events/guildScheduledEventCreate');
+const interactionCreateEvent = require('./events/interactionCreate');
+const pollVotationResultEvent = require('./events/pollVotationResults');
+const qaMetionEvent = require('./events/qaMention');
+const readyEvent = require('./events/ready');
 
-module.exports = (client) => {
-  const eventsPath = path.join(__dirname, 'events');
-  const eventFiles = fs
-    .readdirSync(eventsPath)
-    .filter((file) => file.endsWith('.js'));
-
-  for (const file of eventFiles) {
-    const event = require(path.join(eventsPath, file));
+const eventList = [
+  guildScheduledEventCreate,
+  interactionCreateEvent,
+  pollVotationResultEvent,
+  qaMetionEvent,
+  readyEvent,
+];
```

Comment:
Same comment here:

The list is big, and it's weird to have to enter in this file to add or remove commands (since it has some core logic). What about creating this list in another file and only importing the list here? so this file doesn't have to be changed every time we want to add a new command.